.github/workflows/gcr-cleaner.yaml

```
@@ -0,0 +1,32 @@
+# .github/workflows/gcr-cleaner.yaml
+name: 'gcr-cleaner'
```

```suggestion
name: GCR Cleanup
```

To match our other GitHub Actions names (Containers, Staging Containers, etc.) could we rename the action?